### PR TITLE
Cleanup unused files, improve build infra, add license to postgres, etc.

### PR DESCRIPTION
**Pull request by synackd**
Clean up a lot of the unused files in the repository that are no longer needed. Also, improve the build infrastructure such as streamlining the Makefile and Github Actions config. Further, add the Triad license to the included postgres package.

Many of the deleted files are integration tests that have moved to [deployment-recipes](https://github.com/OpenCHAMI/deployment-recipes).

Is it worth keeping things like:

- scripts/doVersionRelease.sh
- runSnyk.sh
- dev/

?

### CODE REVIEW

**Comment by davidallendj** [comment]
Personally, I think if we don't use it or need it, it's probably best to get rid of it. I didn't see anything in any of the things listed  that should impact BSS working like it is already.

**Review by davidallendj — APPROVED** [review]
LGTM.

File: Makefile [hunk 1]
```diff
@@ -22,36 +22,38 @@
...
+
+# Version
+BUILD := $(shell git rev-parse --short HEAD)
+VERSION := $(shell git describe --tags --always --abbrev=0)
 BINARIES = boot-script-service bss-init
 GOOS := $(if $(GOOS),$(GOOS),linux)
 GOARCH := $(if $(GOARCH),$(GOARCH),amd64)
 
+.PHONY: all
+all: binaries
```
[the remark this answers is not in the document]
Comment (synackd) [reply Makefile:35]: Is that the expected behavior? This target is run when running `make` without any targets passed. Usually I would think that that would make the binaries, but are we wanting to build the container by default?

[the remark this answers is not in the document]
Comment (davidallendj) [reply Makefile:35]: Actually, I think that would make more sense. Let's keep it that way.

**Comment by synackd** [comment]
Just pushed some commits that deletes the items in question in the initial comment. If we decide we need these, we can always refer to them or revert them.

**Review by davidallendj — APPROVED** [review]
LGTM.

**Pull request merged**